validator/src/aggregation/update.rs

```diff
@@ -0,0 +1,43 @@
+use nimiq_handel::{contribution::AggregatableContribution, update::LevelUpdate};
+use serde::{Deserialize, Serialize};
+
+/// The serializable/deserializable representation of a LevelUpdate. It does away with the origin,
```

Comment:
```suggestion
/// The serializable/deserializable representation of a LevelUpdate. It goes away with the origin,
```

Reply:
`does` is correct. But if you want to reword it, we could use "It skips teg origin,"

Reply:
I have changed it for a more straight forward wording.